openupgrade_scripts/scripts/l10n_it/15.0.0.3/pre-migration.py

```diff
@@ -0,0 +1,24 @@
+from openupgradelib import openupgrade
+
+
+@openupgrade.migrate()
+def migrate(env, version):
+    to_delete_xml_ids = [
+        "l10n_it.tax_group_iva_10_de_50",
+        "l10n_it.tax_group_iva_20_de_10",
+        "l10n_it.tax_group_iva_20_de_15",
+        "l10n_it.tax_group_iva_20_de_40",
+        "l10n_it.tax_group_iva_20_de_50",
+        "l10n_it.tax_group_iva_21_de_10",
+        "l10n_it.tax_group_iva_21_de_15",
+        "l10n_it.tax_group_iva_21_de_40",
+        "l10n_it.tax_group_iva_21_de_50",
+        "l10n_it.tax_group_iva_21_inde",
+        "l10n_it.tax_group_iva_22_de_10",
+        "l10n_it.tax_group_iva_22_de_15",
+        "l10n_it.tax_group_iva_22_de_40",
+        "l10n_it.tax_group_iva_22_de_50",
+        "l10n_it.tax_group_iva_22_inde",
+        "l10n_it.tax_group_iva_4_de_50",
+    ]
+    openupgrade.delete_records_safely_by_xml_id(env, to_delete_xml_ids)
```

Comment:
Also, why delete them? Cannot ORM handle it? They are not `noupdate` records.

Reply:
Also, why in pre-migration (instead of post-migration)?

Reply:
> Also, why in pre-migration (instead of post-migration)?

Fixed in https://github.com/OCA/OpenUpgrade/pull/5098/commits/da3816994ae2f16bcfdaebfdfda510bf1e022ff8